webapp/app/js/perf.js

```diff
@@ -28,11 +28,15 @@ perf.factory('seriesSummary', ['seriesColors', function(seriesColors) {
     if (signatureProps.job_group_symbol === "T-e10s") {
       extra = " e10s";
     }
-    var testName = signatureProps.suite + " " + signatureProps.test +
+    var testName = signatureProps.test;
+    if (testName === undefined)
+      testName = "summary";
```

Comment:
it seems to be common in this code to use {} even if there is only one line of code following it.  Can we add that for this if condition?